s4e-web/cypress/page-objects/map/map-locations-search.po.ts

```diff
@@ -89,31 +88,31 @@ export class LocationsSearch extends Core {
       .should('not.be.visible');
 
     return LocationsSearch;
-  }
+  };
 
   static clearSearch() {
     LocationsSearch
       .pageObject
       .getClearSearchBtn()
-      .click({ force: true });
+      .click();
 
     return LocationsSearch;
-  }
+  };
 
   static selectActiveResult() {
     LocationsSearch
       .pageObject
       .getSelectActiveResultBtn()
-      .click({ force: true });
+      .click();
 
     return LocationsSearch;
-  }
+  };
 
-  static waitForCitiesList() {
+  static waitForCitiesList(name:string) {
```

Comment:
You could merge methods `type` and `waitorCitiesList` into one `searchCitiesBy(searchValue: string)`

Reply:
I use .type() for type {enter} key too so I can't merge it